benchmarks/torchbench_model.py

```diff
@@ -209,18 +209,20 @@ def apply_default_precision_config(self, test, benchmark):
     elif test == "train" and hasattr(benchmark, 'DEFAULT_TRAIN_CUDA_PRECISION'):
```

Comment:
Can you add a comment that this is a workaround and should really be fixed in the bridge?

Reply:
Done in https://github.com/pytorch/xla/pull/6139/commits/2791a22abdf55a30766c0b9f343612a386cee532.

Reply:
@frgossen @golechwierowicz Why do we need to set an environment variable for that? Is it related to https://github.com/pytorch/pytorch/issues/115792?